build/SolutionWideAnalyzerConfig.ruleset

```diff
@@ -135,8 +78,62 @@
     <Rule Id="CA2244" Action="None" />
     <Rule Id="CA9999" Action="None" />
   </Rules>
-  <Rules AnalyzerId="Text.CSharp.Analyzers" RuleNamespace="Text.CSharp.Analyzers">
-    <Rule Id="CA1704" Action="None" />
+  <Rules AnalyzerId="Microsoft.CodeQuality.CSharp.Analyzers" RuleNamespace="Microsoft.CodeQuality.CSharp.Analyzers">
+    <Rule Id="Async001" Action="None" />
+    <Rule Id="Async002" Action="None" />
+    <Rule Id="Async003" Action="None" />
+    <Rule Id="Async004" Action="None" />
+    <Rule Id="Async005" Action="None" />
+    <Rule Id="Async006" Action="None" />
+    <Rule Id="CA1001" Action="None" />
+    <Rule Id="CA1003" Action="None" />
+    <Rule Id="CA1019" Action="None" />
+    <Rule Id="CA1032" Action="None" />
+    <Rule Id="CA1065" Action="None" />
+    <Rule Id="CA1200" Action="None" />
+    <Rule Id="CA1500" Action="None" />
+    <Rule Id="CA1507" Action="None" />
+    <Rule Id="CA1726" Action="None" />
+    <Rule Id="CA1821" Action="None" />
+    <Rule Id="CA2001" Action="None" />
+    <Rule Id="CA2109" Action="None" />
+    <Rule Id="CA2200" Action="None" />
+    <Rule Id="CA2234" Action="None" />
+  </Rules>
+  <Rules AnalyzerId="Microsoft.NetCore.Analyzers" RuleNamespace="Microsoft.NetCore.Analyzers">
+    <Rule Id="CA1304" Action="Error" />
+    <Rule Id="CA1305" Action="None" />
```

Comment:
Did you have to disable CA1305? The FxCop on the build machine will fail. What is the offending code?

Reply:
I can revert this now.  I had a bug with using the same instance of token cache with user and app cache and I added a Guid.NewGuid().ToString("N") as a member to debug it, and it complained that I needed ToString("N", CultureInfo.InvariantCulture) but that signature doesn't exist on all platforms so I turned it off.  I'll turn it back on now.

---
In reply to: [241474567](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/723#discussion_r241474567) [](ancestors = 241474567)